playbooks/terraform-provider-openstack-acceptance-test-telefonica/run.yaml

```diff
@@ -0,0 +1,44 @@
+- hosts: all
+  become: yes
+  roles:
+    - config-golang
+    - role: export-cloud-openrc
+      vars:
+        cloud_name: 'telefonica'
+  tasks:
+    - name: Run acceptance tests with terraform-provider-openstack against telefonica public cloud
+      shell:
+        cmd: |
+          apt-get install python-pip -y
+          pip install -U python-openstackclient
+
+          openstack flavor create m1.acctest --id 99 --ram 512 --disk 5 --vcpu 1 --ephemeral 10
+          openstack flavor create m1.resize --id 98 --ram 512 --disk 6 --vcpu 1 --ephemeral 10
```

Comment:
are you sure we can create flavor ?

Reply:
Sorry, I can't see that, @h00130372 please commit new PR to fix @liu-sheng 's comment.